specification/cost-management/resource-manager/Microsoft.CostManagement/preview/2022-02-01-preview/scheduledActions.json

```diff
@@ -0,0 +1,1004 @@
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          },
+          {
+            "name": "checkNameAvailabilityRequest",
+            "description": "Scheduled action to be created or updated.",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/CheckNameAvailabilityRequest"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Response of scheduled action name availability.",
+            "schema": {
+              "$ref": "#/definitions/CheckNameAvailabilityResponse"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    },
+    "/{scope}/providers/Microsoft.CostManagement/checkNameAvailability": {
+      "post": {
+        "tags": [
+          "ScheduledActions"
+        ],
+        "operationId": "ScheduledActions_CheckNameAvailabilityByScope",
+        "description": "Checks availability and correctness of the name for a scheduled action within the given scope.",
+        "x-ms-examples": {
+          "ScheduledActionCheckNameAvailabilityByScope": {
+            "$ref": "./examples/checkNameAvailability-shared-scheduledAction.json"
+          }
+        },
+        "parameters": [
+          {
+            "$ref": "#/parameters/scopeParameter"
+          },
+          {
+            "$ref": "#/parameters/apiVersionParameter"
+          },
+          {
+            "name": "checkNameAvailabilityRequest",
+            "description": "Scheduled action to be created or updated.",
+            "in": "body",
+            "required": true,
+            "schema": {
+              "$ref": "#/definitions/CheckNameAvailabilityRequest"
+            }
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "Response of scheduled action name availability.",
+            "schema": {
+              "$ref": "#/definitions/CheckNameAvailabilityResponse"
+            }
+          },
+          "default": {
+            "description": "Error response describing why the operation failed.",
+            "schema": {
+              "$ref": "#/definitions/ErrorResponse"
+            }
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "CheckNameAvailabilityRequest": {
+      "type": "object",
+      "description": "The scheduled action check name availability request.",
+      "properties": {
+        "name": {
+          "type": "string",
+          "description": "Scheduled action name."
+        },
+        "type": {
+          "type": "string",
+          "description": "Resource type. This is expected to be Microsoft.CostManagement/ScheduledActions."
```

Comment:
Is this case-sensitive or not? E.g. will it work if input `Microsoft.CostManagement/scheduledActions`

Reply:
It is not case-sensitive.